kapitan/version.py

```diff
@@ -8,7 +8,7 @@
 """Project description variables."""
 
 PROJECT_NAME = "kapitan"
-VERSION = "0.32.0"
+VERSION = "0.33.1"
```

Comment:
this is the main change in this PR.

the latest release shows an incorrect version

```
$ docker run -t --rm kapicorp/kapitan:0.33.0 --version
0.32.0
```

```